src/bremsstrahlung.cpp

```diff
@@ -65,15 +65,15 @@ void thick_target_bremsstrahlung(Particle& p, double* E_lost)
   double y = std::exp(y_l + (y_r - y_l)*f);
 
   // Sample number of secondary bremsstrahlung photons
-  int n = y + prn();
+  int n = y + prn(p.prn_seeds_ + p.stream_);
```

Comment:
What do you think about adding a member function the gets the current seed, something like:
```C++
uint64_t* Particle::current_seed() { return p.prn_seeds_ + p.stream_; }
```
which would just make this call:
```C++
int n = y + prn(p.current_seed());
```